docs-kits/kits/Quality-Kit/Software Development View/page_software-development-view.md

```diff
@@ -11,7 +11,93 @@ sidebar_position: 4
+| `https://purl.org/dc/terms/description`            | `<whatever>`                                                     | This property is QM-specific. For human-readable content, rdfs:comment is the usual property but would introduce another namespace so the dct-native property is chosen here.                                                                                                                                                                                                                                                   |
+| `https://purl.org/dc/terms/conformsTo`             | `{"@id":"<urnOfTheCorrespondingAspectModel>"}`                   | This property is QM-specific. It holds the exact aspect-model-URN that defines the schema of the presented dataset including its version. The version in here refers to the data model's version while the EDC-property `cx-common:version` defines the version of the underlying API serving the data.                                                                                                                         |
+| `http://www.w3.org/ns/dcat#qualifiedRelation`      | `{"dct:isPartOf": {"@id": "<idOfTheCorrespondingQualityTask>"}}` | This property is QM-specific. All Asset types defined in this Kit must include this property as it links the data behind an asset with the correct QualityTask. Note that the id of the QualityTask must be used, not the id of the EDC-Asset shielding said QualityTask.                                                                                                                                                       |
+| `https://w3id.org/edc/v0.0.1/ns/type`              | `AmazonS3`                                                       | This property signifies the EDC dataplane that the QM data will be transferred over. The expectation that this would be signaled via the dcat:DataSet-dcat:distribution property of the catalog currently isn't implemented in the EDC. Thus the data must be replicated here and is presented via the same property that the consumer-side `transferprocesses` API uses for this same signal.                                  |
+| `https://w3id.org/catenax/ontology/common#version` | `"1.0"`                                                          | CX-0018 recommends to use cx-common:version to signify the API's version. Since QM has a tight connection between the API and the datamodel, this value could describe the version of the CX-API-standard for the Quality use-case. Creation is currently in progress as CX-0123 v1.0.0.                                                                                                                                        |
```

Comment:
Version is fine, but I think we need to make sure that the Quality KIT clearly states what versions are included in the current release and which versions should be use. As of now, this information is missing, I think.

Reply:
cx-common:version is the version of CX-0123.
CX-0123 will likely link to CX-0018 v2.1.0.
CX-0018 demands DSP 0.8.

I don't see any trouble here. Should I explain the logical reason above in the Kit?

Reply:
Ok, understood and fine for me.